mongodb/src/mongodb/client.bal

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/java;
+
+# Represents the MongoDB client.
+public type Client client object {
+
+    handle datasource;
+
+    public function __init(ClientConfig config) returns ApplicationError? {
+        self.datasource = check initClient(config);
+    }
+
+    # Lists the database names in the MongoDB server.
+    #
+    # + return - An array of database names on success or else returns an error
```

Comment:
```suggestion
    # + return - An array of database names on success or else returns an error
```
Do we need to specify the error?

Reply:
Yes as there can be error due t network when trying t connect t the DB Improved doc comment in c8145ff